bin/crew

```diff
@@ -652,6 +652,24 @@ def upgrade
   end
 end
 
+def download_file(url, filename = url.split('/')[-1])
```

Comment:
Be notice that if the function is defined in the main script, it cannot called by the package recipe (put it in a separate file and `require` it in package.rb)

Reply:
really? I didn't have a problem with it. just call the function `download_file`

Reply:
Oh, looks like I am wrong😅